src/LowerTriangularMatrices/lower_triangular_matrix.jl

```diff
@@ -414,8 +414,41 @@ Base.similar(L::LowerTriangularArray{T,N,ArrayType}, ::Type{T}) where {T, N, Arr
 Base.similar(L::LowerTriangularArray{T}) where T = similar(L, T)
  
 # ARITHMETIC
+# While these would work just via broadcast (and the abstractarray interface), these explicit definitions are significantly faster
+# only mul/div with scalar and addition/subtraction, others are handled via the custom broadcast
+function Base.:(*)(L::LowerTriangularArray{T}, s::Number) where T
+    sT = convert(T, s)
```

Comment:
I don't think that any of the `convert` are necessary because they should be handled upstream in `L.data * s`